engines/ags/shared/gfx/allegro_bitmap.cpp

```diff
@@ -123,13 +123,42 @@ void Bitmap::Destroy() {
 	_isDataOwner = false;
 }
 
+BITMAP *Bitmap::ConvertToAllegroFormat(BITMAP *al_bmp) {
+	Graphics::ManagedSurface *origSurface = new Graphics::ManagedSurface(al_bmp->getSurface());
```

Comment:
This should be:
`const Graphics::ManagedSurface &origSurface = al_bmp->getSurface()`
This avoids uneeded copy of the surface.